src/ReleaseHistory.md

```diff
@@ -6,11 +6,14 @@ Now that an underlying bug in `PropertyBagConverter` has been fixed, there is no
 * FEATURE: Expanding Sarif SDK query mode to support Result.Uri, string StartsWith/EndsWith/Contains.
 * FEATURE: Adding Result.Run and a populating method, so that methods which need the Run context for a given Result have an integrated way to retrieve it.
 * FEATURE: Enhanced property bag serialization unit testing. [#1673](https://github.com/microsoft/sarif-sdk/issues/1673)
```

Comment:
>FEATURE [](start = 2, length = 7)

Now I'm confused. Michael successfully published 2.1.19 last week, and the release history in that NuGet package looks like this:
```
## **v2.1.19** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.1.19) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.1.19) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.1.19) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.1.19)
* API BREAKING: Remove helper method `SarifUtilities.DeserializeObject` introduced in 2.1.15 to fix [#1577](https://github.com/microsoft/sarif-sdk/issues/1577).
Now that an underlying bug in `PropertyBagConverter` has been fixed, there is no need to work around it with this helper method. `JsonConvert.DeserializeObject` works fine.
* FEATURE: Expanding Sarif SDK query mode to support Result.Uri, string StartsWith/EndsWith/Contains.
* FEATURE: Adding Result.Run and a populating method, so that methods which need the Run context
for a given Result have an integrated way to retrieve it.
```
That is, it contains the first three lines above (the API BREAKING and the first two FEATUREs). Please rework to release history to include that 2.1.19, and push the rest up to 2.1.20.

Reply:
Will do.